pygmt/tests/test_filter1d.py

```diff
@@ -0,0 +1,96 @@
+"""
+Tests for filter1d.
+"""
+
+import os
+
+import numpy as np
+import pandas as pd
+import pytest
+from pygmt import filter1d
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import GMTTempFile
+from pygmt.src import which
+
+
+@pytest.fixture(scope="module", name="data")
+def fixture_table():
+    """
+    Load the grid data from the sample earth_relief file.
+    """
+    fname = which("@MaunaLoa_CO2.txt", download="c")
+    data = pd.read_csv(
+        fname, header=None, skiprows=1, sep="\s+", names=["date", "co2_ppm"]
```

Comment:
> The style checks are getting stuck on the argument ("\s+") passed to the `sep=` parameter for `pd.read_csv`. As far as I can tell, that is the correct way to be importing the MaunaLoa_CO2.txt data file and reading it into a dataframe. Is there a way to get this style check to pass?

Try using a raw-string.
```suggestion
        fname, header=None, skiprows=1, sep=r"\s+", names=["date", "co2_ppm"]
```
Similar to here:
https://github.com/GenericMappingTools/pygmt/blob/4b80f3c269f39917c8bb7ee96c5a253279877024/pygmt/datasets/samples.py#L26

Reply:
Didn't realize I could pass a raw string in as an argument. Thanks!